.github/workflows/ci-tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         posargs: ["--xvfb-backend xvfb --durations=10"]
         include:
           - version: "py312"
-            coverage: "--cov-report=xml --cov=geovista"
+            coverage: "--cov-report= --cov=geovista"
```

Comment:
There is actually no need to generate an `XML` report, but the `.coverage` data file is still available - just a minor saving.